tests/integration/test_stages.py

```diff
@@ -806,7 +808,9 @@ def test_block_renders(self):
         Ensure block shows team members.
         """
         user_id = 1
-        self.project_api_mock.get_user_roles_for_course = mock.Mock(return_value=[{'role': ALLOWED_OUTSIDER_ROLES[0]}])
+        self.project_api_mock.get_user_roles_for_course = mock.Mock(return_value=[
+            {'role': AuthXBlockMixin._DEFAULT_TA_ROLE[0]}
```

Comment:
@e-kolpakov  I'd rather suppress it locally, as ``_DEFAULT_TA_ROLE`` for me is a implementation detail. Let me know if it's OK with you.